src/components/node-list/node-list-row-list.js

```diff
@@ -122,11 +78,4 @@ const NodeRowLazyList = ({
   </LazyList>
 );
 
-export const mapStateToProps = (state, ownProps) => ({
-  lazy: state.flags.lazy,
-  ...ownProps,
-});
-
-export default connect(mapStateToProps)((props) =>
-  props.lazy ? <NodeRowLazyList {...props} /> : <NodeRowList {...props} />
-);
+export default NodeRowLazyList;
```

Comment:
Given that the Lazy list is now the default behaviour of this component, I would suggest renaming this function to match the name of the file as referenced in other parts of the app (i.e `NodeRowList` as imported in `node-list-group.js`) for clarity.